packages/mermaid/src/diagrams/packet/parser.ts

```diff
@@ -0,0 +1,14 @@
+import type { Packet } from 'mermaid-parser';
+import type { ParserDefinition } from '../../diagram-api/types.js';
+
+import { parse } from 'mermaid-parser';
+import { log } from '../../logger.js';
+import { populate } from './db.js';
```

Comment:
Why did you implement `populate` inside `db.ts`?

Reply:
Moved